### DISCUSSION TITLE
Force rendering at actuators locations

### DISCUSSION THREAD

**Discussion by rohit-kumar-j** [google-deepmind/mujoco#495] [Asking for Help]
For rendering force at actuators positions, in [this function](https://github.com/rohit-kumar-j/mujoco-python-viewer/blob/main/mujoco_viewer/mujoco_viewer.py#L265), it uses `site.xmat` and `site.xpos` by placing sites at actuators locations. Is there an alternative to get the `xpos` and `xmat` of the actuators something like:

```python
data.joint("joint_name").location_pos   # (nu x 3)
data.joint("joint_name").location_mat   # (nu x 9)
```

[This is the example used:](https://github.com/rohit-kumar-j/mujoco-python-viewer/blob/main/examples/actuator_force_rendering.py)
<img src="https://user-images.githubusercontent.com/37873142/191242981-3e3300c2-69e9-4153-92e8-d4aa4c8b796d.png" width="700" />

**Comment by yuvaltassa** [comment]
`data.joint("joint_name").xanchor` is the global joint position.
`data.joint("joint_name").xaxis` is the global joint axis orientation (so 3, not 9).
This is all reasonably well documented in the comments [here](https://github.com/deepmind/mujoco/blob/main/include/mujoco/mjdata.h#L203), but I agree that the removal of the `jnt_` and `body_` prefixes for some fields is annoying and makes some quantities harder to discover.

**Comment by yuvaltassa** [comment]
PS, please use issues rather than discussions. It's a bit annoying that questions can come up in two places and we're trying to move away from discussions for things that are not actually discussions (e.g. questions like this one)

**Reply by rohit-kumar-j** [reply]
Certainly
